packages/babel-core/src/config/full.ts

```diff
@@ -113,6 +113,7 @@ export default gensync(function* loadFullConfig(
 
       for (let i = 0; i < rawPresets.length; i++) {
         const descriptor = rawPresets[i];
+        // @ts-expect-error TODO: disallow false
```

Comment:
What do you think?
Currently Babel accepts `["plugin", false]` to disable a plugin, is there any reason for this?